contrib/pinot-druid-migration/HEADER

```diff
@@ -0,0 +1,16 @@
+Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This file may not needed anymore since we removed `contrib` from checking license header.